Interop/CoreDiagnostics/Public/FIRCoreDiagnosticsData.h

```diff
@@ -19,36 +19,34 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /** If present, is a BOOL wrapped in an NSNumber. */
-static NSString *const kFIRCDIsDataCollectionDefaultEnabledKey =
-    @"FIRCDIsDataCollectionDefaultEnabledKey";
+#define kFIRCDIsDataCollectionDefaultEnabledKey @"FIRCDIsDataCollectionDefaultEnabledKey"
 
 /** If present, is an int32_t wrapped in an NSNumber. */
-static NSString *const kFIRCDConfigurationTypeKey = @"FIRCDConfigurationTypeKey";
+#define kFIRCDConfigurationTypeKey @"FIRCDConfigurationTypeKey"
```

Comment:
Are you able to measure any actual reduction in size using this mechanism? This also results in the value being inlined into every translation, but just as a string literal rather than a static variable.

The way to do this within the language is to declare these here, like so:

```objc
extern NSString *const kFIRCDConfigurationTypeKey;
```
Then in the associated .m file:
```objc
NSString *const kFIRCDConfigurationTypeKey = @"FIRCDConfigurationTypeKey";
```

Reply:
The linker should consolidate the string usages. However, in this case there's only one usage in each library, so no need. This change keeps the strings from being added to all the other files including where the strings aren't being used.  The extern methodology doesn't work across optional interop pod dependencies.